collector/mssql.go

```diff
@@ -1862,7 +1890,8 @@ type win32PerfRawDataSQLServerAccessMethods struct {
 	Usedtreepagecookie            uint64
 	WorkfilesCreatedPersec        uint64
 	WorktablesCreatedPersec       uint64
-	WorktablesFromCacheRatio      uint64
+	WorktablesFromCacheHits       uint64
+	WorktablesFromCacheLookups    uint64
```

Comment:
Is this a find/replace mistake? The underlying counters are `WorktablesFromCacheRatio` and `WorktablesFromCacheRatio_Base`, right?

Reply:
You are correct